jina/clients/__init__.py

```diff
@@ -4,25 +4,42 @@
 from .mixin import PostMixin
 from .request import GeneratorSourceType
 from .websocket import WebSocketClientMixin
+from ..parsers import set_client_cli_parser
 
-__all__ = ['Client', 'WebSocketClient']
+__all__ = ['Client', 'GrpcClient', 'WebSocketClient']
 
 
-class Client(PostMixin, BaseClient):
+def Client(host: str, port_expose: int, restful: bool = False) -> 'BaseClient':
+    """Jina Python client.
+
+    :param host: Host address of the flow.
+    :param port_expose: Port number of the flow.
+    :param restful: If connect to a Restful gateway, default is ``False``, connect to GrpcGateway.
+    :return: An instance of :class:`GrpcClient` or :class:`WebSocketClient`.
+    """
+    args = set_client_cli_parser().parse_args(
+        ['--host', host, '--port-expose', str(port_expose)]
+    )
```

Comment:
when written in this way, all other client arguments are not passed. better use `**kwargs` and use `ArgNamespace.kwargs2list()`
```
usage: jina client [-h] [--request-size] [--continue-on-error]
                   [--show-progress] [--return-results] [--host]
                   [--port-expose] [--proxy]

Start a Python client that connects to a remote Jina gateway

⚙️  Optional Arguments

  -h, --help           show this help message and exit

⚙️  Client Arguments

  --request-size       type: int; default: 100

                       The number of Documents in each Request.

  --continue-on-error  default: disabled, use "--continue-on-error" to
                       enable it

                       If set, a Request that causes error will be logged only
                       without blocking the further requests.

  --show-progress      default: disabled, use "--show-progress" to enable
                       it

                       If set, client will show a progress bar on receiving
                       every request.

  --return-results     default: disabled, use "--return-results" to enable
                       it

                       This feature is only used for AsyncClient.

                       If set, the results of all Requests will be returned as
                       a list. This is useful when one wants process Responses
                       in bulk instead of using callback.

⚙️  Expose Arguments

  --host               type: str; default: 0.0.0.0

                       The host address of the runtime, by default it is
                       0.0.0.0.

  --port-expose        type: int; default: 58570

                       The port of the host exposed to the public

  --proxy              default: disabled, use "--proxy" to enable it

                       If set, respect the http_proxy and https_proxy
                       environment variables. otherwise, it will unset these
                       proxy variables before start. gRPC seems to prefer no
                       proxy
```